ci_cd.sh

```diff
@@ -59,12 +62,15 @@ if [ $large_test = true ]; then
 fi
 
 # Compute image tag
-org_name=fabernovel
-simple_image_name=api-$android_api
+org_name="fabernovel"
+simple_image_name="api-$android_api"
+if [[ "$gcloud" = true ]]; then
+  simple_image_name="$simple_image_name-gcloud"
+fi
 if [ "$android_ndk" = true ]; then
   simple_image_name="$simple_image_name-ndk"
 fi
-branch=${GIT_REF##refs/heads/}
+branch="${GIT_REF##refs/heads/}"
```

Comment:
Is this change needed? For best practice perhaps?

Reply:
https://google.github.io/styleguide/shellguide.html#quoting

Reply:
Great guide! But we must be sure its ok with `#!/bin/sh` (the guide is for `bash`).

I should open an issue to see at which point we can go back to bash (was something related with GitHub Action container, but it might be obsolete now).

Reply:
#70